packages/gatsby-remark-images-contentful/src/utils/index.js

```diff
@@ -1,20 +1,34 @@
 const axios = require(`axios`)
 
-const getBase64Img = async url => {
-  const response = await axios({
-    method: `GET`,
-    responseType: `arraybuffer`,
-    url: `${url}`,
-  })
+const getBase64Img = async (url, reporter) => {
+  let base64Img
+  try {
+    const response = await axios({
+      method: `GET`,
+      responseType: `arraybuffer`,
+      url: `${url}`,
+    })
 
-  const base64Img = `data:${
-    response.headers[`content-type`]
-  };base64,${new Buffer(response.data).toString(`base64`)}`
+    base64Img = `data:${response.headers[`content-type`]};base64,${new Buffer(
+      response.data
+    ).toString(`base64`)}`
+  } catch (err) {
+    reporter.panic(`Failed downloading the base64 image for ${url}`, err)
+  }
```

Comment:
I would return inside the `try` and `catch` blocks explicitly.

```suggestion
    return `data:${response.headers[`content-type`]};base64,${new Buffer(
      response.data
    ).toString(`base64`)}`
  } catch (err) {
    reporter.panic(`Failed downloading the base64 image for ${url}`, err)
    return undefined
  }
```